CONTRIBUTING.md

```diff
@@ -13,14 +13,77 @@ Please fill out either the individual or corporate Contributor License Agreement
 
 Follow either of the two links above to access the appropriate CLA and instructions for how to sign and return it. Once we receive it, we'll be able to accept your pull requests.
 
+## Composition of This Repository and Where/How to Contribute
+
+The repository of the Kubernetes Python client consists of this main repository and a submodule, the [python-base](https://github.com/kubernetes-client/python-base) repository. The main repository contains mostly files that are generated by the OpenAPI generator from [this OpenAPI spec](scripts/swagger.json). The base repo is the utility part of the python client and allows developers to create their own kubernetes clients.
```

Comment:
"The repository of the Kubernetes Python client consists of this main repository" might be better as "The  Kubernetes Python client consists of this main repository, the [python](https://github.com/kubernetes-client/python)"

Also in the last sentence, by "base repo" I think you mean python-base which you refer to above as "submodule". It might be clearer to someone new if you keep the terms consistent.